src/Services/MemberServices.js

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
```

Comment:
this is members service. Not categories service. Please update the right service.